### ISSUE TITLE
Rename airhorn.wav to airhorn.mp3

### ISSUE THREAD

**Issue by cisoun** [nichuguen/airhorn.club#1]
Hello,

I'm really TRIGGERED because I have found an error in your HTML page so I'm here to complain because my life is so empty (pls kill me) and fak u gooby.

Here's my issue:
In https://github.com/ymaktepi/airhorn.club/blob/master/app/index.html:8, you are defining the following text as your page's title : "airhorn.wav". Ok so far. BUT THEN, at line 13, you are actually linking the file `airhorn.mp3` **WHICH IS A MP3 FILE !! CAN YOU READ THE EXTENSION ?!** My point is: **WTF MAN IT'S FRICKING CONFUSING**

# WHY
# ARE
# YOU
# LYING

FFS I swear I'm calling the cops/REGA/Coop Pronto/Jean-Marc Richard right now....

Okay now please fix that yourself. I don't want to waste my time on such issue created by a cis white privileged man like you... Get your sh!t together dude.

Have a nice day.

Regards,

Dolan Duck
CTO on Marmiton.org
dolan.duck@ur-mom-gaey.io

**Comment by ymaktepi** [comment]
ok dolan

**Comment by ymaktepi** [comment]
@DoubleSentinels ?

**Comment by DoubleSentinel** [comment]
Dear @cisoun ,

We were lucky to get some strong proposals in on this and we deeply appreciate all the info from _he comes_ your side — and for your patience. After a lot of careful **mad** thought, we have decided to go with another _w̴h̸i̴s̵p̸e̵r̵ ̵i̵n̸ ̸d̴a̵r̵k̵n̴e̴s̸s̸_ for this project. While we certainly have no doubt about the superior quality of your _**TRASH**_ or that you could deliver on this skillfully h̶e̶ ̸c̵o̴m̸e̵s̵ w̷͕̉̐ȇ̶̱̑ ̵̢̝̀d̵̖͂͒e̸̯̦̿ĉ̸̣̭i̵̠̟͐̊d̴̛̛͔̖è̶̘ͅd̷̰̏̎ ̵̧̗̔͘t̵̜̀õ̸̯̰͗ ̷̭͉̆͆u̵̮͙͂s̷͍͝͝ě̵̳͠ ̸̞̓ť̷̬̙h̶̬͑ï̸̥ŝ̵͈̤ p̴r̶o̷j̴e̴c̸t̶ ̵t̴o̷ ̵e̵x̴p̴a̴n̸d̵ ̴o̶u̸r̸ ̶r̶e̸a̸c̸h̸ ̵a̸n̸d̴ ̷a̶d̵v̸a̴n̶c̷e̴ ̶o̵u̴r̶ ̶g̶o̵a̷l̷s̵ ̵t̵o̵ ̷t̷h̴e̴ ̵a̷b̸s̸o̷l̷u̶t̷e̶ ̵r̶e̵s̶t̵r̶i̴c̶t̶i̶o̷n̸ ̴o̸f̴ **.̶̡̢͙͙̫͓͓͈͉̠̠̔̔̀͝ͅẄ̴̪̩͔̣͙͉̥̱͖̹͕͓̝͉͖͍̖͉̗̯̤̰̮̻́̄̍͆̍͋͒͊̊̏̐̿̐͂͑͜Ặ̶̧̡̢̧̥̠̣̫͍̟̥̰̣̣̥̖̙̤̩̞̲̞͓̯̰̰͙̰̲̺̝̮̪̣̞͙̹̈̐̏̾̀͊̃̒̍͋́̆́̊͋̐̈́̈́̈́͊͛̈́͒̉̒̅̏͂̋̊͘̚͜͝V̵̢̧̛̛̖͕͕̹͙̖̮͖̞͕͔̩̟͚̫̩͎͖̻̰̰̯͇̼̪̔̾̓̈́̈̊͋̍̏͆͆̃̀̂̋̕͘͝͠** f̸i̸l̵e̷s̴ ̵a̸c̶r̵o̴s̷s̴ _**Ā̸̛̗̳̔̈́̈́̍̉̂̽̎̌̃̋͑̈́̓͊͐́̓̓̊̚̕̚͝L̴̢̛̠͓̰͈̦̓̒̅̌͛͌̕L̸̢̨̢̛̺͖͇̥̺̞̲͎͚̯̞̩̱̼̮̭͚̯̙̝̲̎̊͗͌͑̽̕͘ͅ ̴̧̛̰̬̣̙̫̞͔̗̫̰̋̅̐͆́́͒̀͠P̶̧̡̡̞̩̖̮̲͔͍̜̯͔̺̔́̽̔̈́̿͜͝L̵̢̛̛̦̝̠̬̘͎̳̫̱̭̼̰̹̘̤̜͎̻̪̹̱̣̗̖̣̘͇̾̓̀͊̐̍͂̇̏̔̏̃͋̒̊̍̇͘͘͝A̷̳̠̼̅̏͗͗̏̃̆̈́̽́T̵̛̥̺̲̻̯͉͉̗̣͔̤̣̮̱̠̀̈͗͂͋͂̄̑̄͒́̐̓̄̐̾̐͛̚̚͘̕͠͝Ė̶̱͉͉̘̦͈͔̲̗̘͕̅̍͌̇̉̈͠͝F̸̡̮̳̦̦̩̹̥̙͖̎́̿̊̓̀̿͊Ǫ̶̬̬̽̂͛̇͑͛̇̾͘͠Ŗ̷̧̧̝̜̲̪̙̟̣̯̘̖̰̅̿͗̏̈́̀̌͌̆̾̈́̀̀͂͘M̵̳̩̻̪͔͓͙̖͙̖̉̈͊̎̀͂̽͒̈́͊͗͒͗̀̃͘S̵̨̧̨͇͍͓͎͎̙̯̱͙͍̺̗̣͉͈̗̜̥̺͉͓͈̲̩̥̀̍̈́̊̏̔̆͂͋͒͊̈́̌̏͝͝ͅ.̴̛̛͈̰̤̲̫͙͖͍͆̅̀̌̓̓̐͂̾̌̈̉́̈̔̓̈́̉̌̽̀͠͝**_

We’d really like to continue talking with you about future projects we have coming up this year. I definitely look forward to collaborating in the future.

Thanks again for your help and your time,
Shub-Niggurath, the Whisperer in Darkness

**Comment by cisoun** [comment]
Dear @DoubleSentinel,

Thanks for your "kind" reply (which was IMHO very autistic, how old are you ? 12 ? What's your level on Fortine ? Seriously I'll 1v1 whenever you want, beach !).
So yeah, I've got in touch with Mr. @ymaktepi which apologized for laying such crap and he kindly proposed me to rub my butt for the inconveniance (das gay sh!t but I'll just say "no homo" and we'll be good). I hope you'll do the same because I cannot understand **HOW...THE...FUARK** you both, could have made such a mistake !!!!¨1¨!!

SO anyway, I've reported both of you to the cyber police. You've been goofed boys, consquences will never be the same.

btw, say hi to your moms.

Regards,

Dolan Duck
CTO on Marmiton.org
dolan.duck@ur-mom-gaey.io

---

PS: closing this ticket. Enjoy the turkish toilets in prison retards.
